components/__snapshots__/DiffView.test.js.snap

```diff
@@ -3437,6 +3443,12 @@ exports[`DiffView component Should render diff with comments 1`] = `
                         </a>
                       </div>
                       <div>
+                        <button
+                          class="btn borderless btn-info btn-sm"
+                          style="color: rgb(255, 255, 255);"
```

Comment:
I think you could use className="text-white" instead
https://getbootstrap.com/docs/5.0/utilities/colors/

Reply:
I think this is assigned by the  custom Button component when you assign values to certain properties.
I dont think the component allows you to pass in a `className` prop.

Reply:
The Button component should be updated so it can accept html attributes. Here is an example of materialui library doing so with their button component:

https://github.com/mui/material-ui/blob/902b4b070b88bf6ce713553a7a30d7a4c3e20c0a/packages/mui-material/src/Button/Button.js#L309

Making this change would definitely make things more convenient. For example you could pass `data-testid` which would allow for easier testing.

Also this is actually necessary so that we can be more semantically correct when using button elements too. For example, on the admin lessons page, the `update Lesson` button doesn't have `type=submit`, and currently that isn't possible.

Reply:
Ok that sounds good. Do you think it makes more sense to include in this PR, or create a separate issue for it? Im fine with either approach.

Reply:
Separate PR is better because this PR has its own issue to solve.